jbt-it/neomdb
# Added the  Application Portal welcome page

How to test:
- make sure you are not logged in and navigate to http://localhost:3000/bewerbung
- then change the applicationStart and applicationEnd in Application.tsx so see each varaition of the welcome page
